yellow-paper/docs/rollup-circuits/base_rollup.md

```diff
@@ -29,7 +29,7 @@ class PartialStateReference {
 }
 
 class StateReference {
-    l1_to_l2_message_tree: Snapshot
+    l1_to_l2_messages_tree: Snapshot
```

Comment:
I don't agree on this change. It makes the tree not match the others. Everything else seems to be singular_tree. See the `PartialStateReference` for naming of others as example